packages/flutter_tools/test/general.shard/resident_runner_test.dart

```diff
@@ -30,7 +30,7 @@ import 'package:flutter_tools/src/run_cold.dart';
 import 'package:flutter_tools/src/run_hot.dart';
 import 'package:flutter_tools/src/version.dart';
 import 'package:flutter_tools/src/vmservice.dart';
-import 'package:unified_analytics/src/enums.dart';
+import 'package:unified_analytics/src/enums.dart' show DashEvent; // ignore: implementation_imports
```

Comment:
@eliasyishak (or maybe @christopherfujino): Is it an oversight that unified_analytics doesn't export `DashEvent` and `DevicePlatform` (see below) even though it is used by the flutter_tool? Or is there a reason for this that I could leave as a comment on this `ignore`?

Reply:
That was by design, it was so that clients of the package didn't have to deal with `DashEvent` but instead work with the data class `Event`.

And outside of tests, `DevicePlatform` didn't have any use in production code so didn't want it to cause any confusion

Reply:
I see. Should this code be refactored to not rely on `DashEvent` then and somehow use the `Event` class?

Reply:
It is kinda strange that the `DashEvent` type is used in the public API of the package, but isn't exported: https://github.com/dart-lang/tools/blob/8323b217373091d258e124ea80e50b4ad52c919d/pkgs/unified_analytics/lib/src/event.dart#L11

IMO, if it is part of the public API, it should be exported. I'll file an issue for this and link it here for now.

Reply:
ah good call, yeah i would agree that it makes sense to export it then, that's not the case for `DevicePlatform` though is it?

Reply:
If DevicePlatform is expected to be used in tests, I would expect that it is also exported. I don't think anyone should be doing `src` imports from other packages even in tests.

Reply:
https://github.com/dart-lang/tools/issues/234

Reply:
got it, thanks for filing!

Reply:
You could export it via a separate `testing.dart` library if it really is only expected to be used in tests.